src/pages/Facility/overview.tsx

```diff
@@ -57,14 +59,17 @@ export function FacilityOverview({ facilityId }: FacilityOverviewProps) {
   ];
 
   return (
-    <div className="">
-      <div className="container p-6 mx-auto space-y-8">
+    <Page title="">
+      <div className="container mx-auto space-y-8">
         {/* Welcome Header */}
-        <div className="p-6 bg-white rounded-lg shadow-sm">
+        <div className="rounded-lg">
           <div className="flex items-center gap-4 mb-4">
             <div>
-              <h1 className="text-2xl font-bold text-gray-900">
-                Hey {user?.first_name}
+              <h1 className="text-4xl font-bold text-gray-900">
+                {t("hey_user", {
+                  user:
+                    (user.prefix ? user.prefix + " " : "") + user.first_name,
```

Comment:
consider using util `formatName`, maybe add a logic to only show nickname

Reply:
Didn't make much sense to abstract out, as we are only using at one place 😅